lumigator/frontend/README.md

```diff
@@ -0,0 +1,112 @@
+# Lumigator Frontend
+
+This directory contains the **frontend** code for the **Lumigator** project, built with **Vue 3**, and **Vite**. It is a modular and scalable front-end application designed to interact with the backend API provided by the Lumigator backend.
+
+## Tech Stack
+
+- **Vue 3**: The progressive JavaScript framework for building modern web interfaces.
+- **Vite**: A fast build tool and development server that supports modern JavaScript and hot module replacement (HMR).
+
+## Getting Started
+
+### Prerequisites
+
+Ensure you have the following installed on your system:
+
+- **Node.js** (version 14 or higher)
+- **npm** or **yarn** package manager
+
+### Installation
+
+1. **Clone the repository** (if you haven't already) and navigate to the `frontend` directory:
+
+   ```bash
+   git clone https://github.com/mozilla-ai/lumigator.git
+   cd lumigator/frontend
+   ```
+
+2. **Install dependencies**:
+
+   Using npm:
+
+   ```bash
+   npm install
+   ```
+
+   Or using yarn:
+
+   ```bash
+   yarn install
+   ```
+
+### Development
+
+To run the frontend locally for development purposes, use the following command to start the Vite development server. This will also enable hot-reloading, so changes you make in the code will be reflected instantly in the browser.
+
+```bash
+npm run dev
+```
+
+Or using yarn:
+
+```bash
+yarn dev
+```
+
+### Visit `http://localhost:3000` in your browser. The application runs at this address by default.
```

Comment:
```suggestion
Visit `http://localhost:3000` in your browser. The application runs at this address by default.
```

I don't think this should be a heading, it's the next step in part of the `### Development` section.